Lib/xml/etree/ElementTree.py

```diff
@@ -1256,21 +1256,25 @@ def iterator(source):
                 it.root = root
         finally:
             if close_source:
-                source.close()
+                1#source.close()
```

Comment:
Unintended change?

```suggestion
                source.close()
```

Reply:
Oh, it was for debugging. But since the tests are passed, I wonder how can we test that this line is present.